docs/pages/project/contributing/contributing-error.md

```diff
@@ -105,7 +105,8 @@ New
 
 ## Replacing logrus
 
-There already exists an [interface for logger](https://github.com/layer5io/meshkit/blob/master/logger/logger.go) in MeshKit.
+There already exists an [interface for logger](https://github.com/layer5io/meshkit/blob/master/logger/logger.go) in MeshKit.<br><br>
+**WARNING**: To enforce the use of meshkit errors, meshkit logger was designed such that it only works with meshkit errors. If a non-meshkit error is logged through the logger, it would panic and kill the process. See: [meshkit#119](https://github.com/layer5io/meshkit/pull/119) for more insight.
```

Comment:
```suggestion
**WARNING**: To enforce the use of meshkit errors, meshkit logger was designed such that it only works with meshkit errors. If a non-meshkit error is logged through the logger, it would panic and kill the process. See: [example](https://github.com/layer5io/meshkit/pull/119) for more insight.
```
writing example might be more helpful than giving a PR no. imo

Reply:
@warunicorn19 I agree.
However this is not exactly an example. Its link to a discussion on why we aren't patching it on meshkit's side, just for reference. : )

Reply:
is `panic` the correct word to use here xd ?
I was thinking of something like: `it would not recognize the error and hence it will kill the process`

Reply:
ohh okay got you @DelusionalOptimist. @Mr-DG-Wick no I think panic is best suited here.

Reply:
ohh cool.

Reply:
Thank you for helping out here folks. :+1: